detox/src/utils/exec.js

```diff
@@ -164,22 +163,33 @@ function _joinCommandAndFlags(command, flags) {
   return result;
 }
 
-async function interruptProcess(childProcessPromise, signal = 'SIGINT') {
-  const log = execLogger.child({ fn: 'interruptProcess' });
+const DEFAULT_KILL_SCHEDULE = {
+  SIGINT: 0,
```

Comment:
@d4vidi , I am wondering whether it makes sense to add extra signals by default here? e.g. SIGTERM after 10s or something like that... 🤔

Reply:
I don't think so. It doesn't feel like the common case. However, maybe a dedicated version of `interruptProcess` that uses a more exotic schedule can be predefined here, and then reused where needed? `interruptProcessFTW()`